src/shellCommands/ocSdkCommands.ts

```diff
@@ -93,20 +96,81 @@ export class OcSdkCommand {
   async runCollectionVerifyCommand(
     outputChannel?: vscode.OutputChannel,
     logPath?: string,
+    namespace?: string,
+    collection: string = "operator_collection_sdk",
   ): Promise<string> {
-    const galaxyUrl = getAnsibleGalaxySettings(AnsibleGalaxySettings.ansibleGalaxyURL) as string;
-    const galaxyNamespace = getAnsibleGalaxySettings(AnsibleGalaxySettings.ansibleGalaxyNamespace) as string;
+    const galaxyUrl = getAnsibleGalaxySettings(
+      AnsibleGalaxySettings.ansibleGalaxyURL,
+    ) as string;
+    const galaxyNamespace =
+      namespace ??
+      (getAnsibleGalaxySettings(
+        AnsibleGalaxySettings.ansibleGalaxyNamespace,
+      ) as string);
     const cmd: string = "ansible-galaxy";
     let args: Array<string> = [
       "collection",
       "verify",
       "-s",
       galaxyUrl,
-      `${galaxyNamespace}.operator_collection_sdk`,
+      `${galaxyNamespace}.${collection}`,
     ];
     return this.run(cmd, args, outputChannel, logPath);
   }
 
+  /**
+   * Installs the "kubernetes.core" collection and "kubernetes" python module if
+   * the collection is not already installed
+   * @param outputChannel - The VS Code output channel to display command output
+   * @param logPath - Log path to store command output
+   * @returns - A Promise containing a boolean signaling the success or failure of the command
+   */
+  async installOcSDKDependencies(
+    outputChannel?: vscode.OutputChannel,
+    logPath?: string,
+  ): Promise<boolean> {
+    try {
+      await this.runCollectionVerifyCommand(
+        outputChannel,
+        logPath,
+        "kubernetes",
+        "core",
+      );
+
+      return true;
+    } catch (e) {
+      const moduleStatusCode = await this.run(
+        "pip",
```

Comment:
We can't assume that the user has `pip` installed, so you should first validate if `pip` or `pip3` is installed in the user's environment.

I'm hitting the following error because I'm currently using `pip3` instead of `pip`.

```
/bin/sh: pip: command not found
```